.vscode/launch.json

```diff
@@ -0,0 +1,19 @@
+{
```

Comment:
I see .vscode is in the .gitignore file. Should the PR exclude those .vscode/*.json files?

Reply:
These are file used from Debugging, Other project like Myla and CCM also have them.
 https://github.com/tl-its-umich-edu/my-learning-analytics/tree/master/.vscode 
https://github.com/tl-its-umich-edu/canvas-course-manager-next/tree/main/.vscode

I have a vscode history plugin installed and it generated the .history file which is of no purpose. 

The files like `launch.json` and `task.json` and actually used in Debugging in VSCODE and stops when you hit a break point.

Reply:
Got it. Thanks!